.idea/codeStyles/Project.xml

```diff
@@ -3,18 +3,9 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="java.util" alias="false" withSubpackages="false" />
```

Comment:
Again I'm not sure if the updated git ignore will apply to this already commited file. In case it won't, please delete it before merging.